src/components/summary-list/index.md

```diff
@@ -68,6 +68,15 @@ If your summary list does not have any actions, you can choose to remove the sep
 
 To remove borders on a single row, use the `govuk-summary-list__row--no-border` class.
 
+### Showing missing information
+
+There may be some contexts where users see a summary list where there is some missing information, for instance if they are returning to a partially-completed form, or if you have added some additional questions since they first answered.
+
+Where there are missing answers, display a link to the question page within the `value` column instead of as a "Change" action:
```

Comment:
Hi @frankieroberto, thanks for the poke, I'm looking at this to get it published. 

How's this look for guidance? I think I've captured what you've been trying to say, also incorporate the 'changed questions' use case in the issue.

I think the main message is to:
- avoid adding another 'change' action to the row
- avoid showing unhelpful status messages, that could also get confused as entered information in the value column,  like "Incomplete" or "Not answered"

```suggestion
### Showing missing information

In some contexts, you might need to show rows that have missing information. This can happen when: - a user returns to an incomplete journey
- you've added or changed the questions in a service.

Show a link to the appropriate question page in the `value` column so the user can enter the missing information, instead of showing a 'change' link on that row.
```

Reply:
Love it!